src/test/java/com/wikia/webdriver/testcases/Anonymization/TestFile.java

```diff
@@ -0,0 +1,10 @@
+package com.wikia.webdriver.testcases.Anonymization;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+public class TestFile
+{
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move this left curly brace to the end of previous line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)